README.md

```diff
@@ -14,22 +14,26 @@ Images are always based on the latest Ubuntu image available on
 ## How to run an image
 
 All images are available in
-[Docker Hub under @schneiderfelipe](https://hub.docker.com/u/schneiderfelipe),
+[GitHub Container Registries](https://github.com/orgs/geem-lab/packages?repo_name=docker-hpc),
 so the command below does everything for you:
 
-    $ docker run -it schneiderfelipe/siesta:4.0.2 bash
+    $ docker run -it ghcr.io/geem-lab/IMAGENAME:TAGNAME /bin/bash
 
 (Of course, you'll need [Docker](https://www.docker.com/).)
 This will give you a minimal, fully functioning terminal with all required
 software available in `/opt`.
 
 *Note*: if you build images locally
 ([see below](#how-to-build-images-locally)), you probably want to remove the
-`schneiderfelipe/` part (see below).
+`ghcr.io/geem-lab/` part (see below).
 
 ## How to build images locally
 
-    $ docker build -t siesta:4.0.2 siesta:4.0.2
+    $ docker build -t IMAGENAME:TAGNAME IMAGENAME:TAGNAME
+
+## How to run images locally
+
+    $ docker run -it IMAGENAME:TAGNAME /bin/bash
```

Comment:
Also here. If the user can't figure out on their own where's the image name and the tag name in a simple example, they don't know Docker enough to run it anyway 😆

Reply:
I have added your suggestion as a direct example, @schneiderfelipe. The best of the both worlds [(MONTANA, Hannah)](https://www.youtube.com/watch?v=uVjRe8QXFHY)